documentation/api/io.strimzi.api.kafka.model.Rack.adoc

```diff
@@ -89,12 +92,23 @@ apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnect
 # ...
 spec:
-  kafka:
-    # ...
-    rack:
-      topologyKey: topology.kubernetes.io/zone
-    # ...
+  # ...
+  rack:
+    topologyKey: topology.kubernetes.io/zone
+  # ...
 ----
 
-Enabling rack awareness in the `KafkaConnect` custom resource will not set any affinity rules, but you can also configure `affinity` or `topologySpreadConstraints`.
-For more information see xref:assembly-scheduling-str[].
+When deploying Mirror Maker 2 using Strimzi, you can use the `rack` section in the `KafkaMirrorMaker2` custom resource to automatically configure the `client.rack` option.
+
+.Example `rack` configuration for Mirror Maker 2
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaMirrorMaker2ApiVersion}
+kind: KafkaMirrorMaker2
+# ...
+spec:
+  # ...
+  rack:
+    topologyKey: topology.kubernetes.io/zone
+  # ...
+----
```

Comment:
What is the actual impact on the different MM2 connectors? Is this something what is desired for all of them?

Reply:
This is mainly for MirrorSourceConnector which creates a consumer inside its task. The other two don't, so it's simply ignored.